ClientServer.go

```diff
@@ -47,7 +46,10 @@ func Server() {
 		// send new string back to client
 		_, sendingerror := conn.Write([]byte(newmessage + "\n"))
```

Comment:
In golang keep local variables name small as possible. Why not just `err` instead of `sendingerror`?
And please use camel case for multi word variables like: `sendingError`